seaborn/utils.py

```diff
@@ -173,9 +173,9 @@ def despine(fig=None, ax=None, top=True, right=True, left=False,
         Specific axes object to despine.
     top, right, left, bottom : boolean, optional
         If True, remove that spine.
-    offset : int or None  (default), optional
+    offset : int or dict of ints or None  (default), optional
         Absolute distance, in points, spines should be moved away
-        from the axes (negative values move spines inward).
+        from the axes (negative values move spines inward). Dict for each side can also be given.
```

Comment:
This line is failing the PEP8 check because it is too long. Also please be more clear about what the dict should look like, i.e. keys should be a spine name and value should be the size of the offset.